modules/src/food.py

```diff
@@ -0,0 +1,123 @@
+import os
+import json
+import requests
+import random
+import difflib
+
+import config
+from templates.generic import *
+from templates.text import TextTemplate
+
+ZOMATO_API_KEY = os.environ.get('ZOMATO_API_KEY', config.ZOMATO_API_KEY)
+
+def process(input,entities):
+	output = {}
+	try:
+		requested_location = entities['restaurant_location'][0]['value']
+
+		header = {'user-key': ZOMATO_API_KEY, 'Accept': 'application/json'}
+		response = requests.get('https://developers.zomato.com/api/v2.1/locations?query=' + requested_location, headers=header)
+		
+		if response.ok:
+			content = response.json()
+			try:
+				location = content['location_suggstions'][0]
+			except (KeyError,IndexError):
+				error_message = 'I couldn\'t find any restaurants matching your query.'
+				error_message += '\nPlease ask me something else, like:'
+				error_message += '\n  - Is Giordano\'s pizza in Chicago good?'
+				error_message += '\n  - Chipotle in Evanston'
+				error_message += '\n  - What are some restaurants in Chicago?'
+				output['error_msg'] = TextTemplate(error_message).get_message()
+				output['success'] = False
```

Comment:
@finleylau  This seems to be duplicated everywhere, Could we modularize the logic, please :)

Reply:
done!